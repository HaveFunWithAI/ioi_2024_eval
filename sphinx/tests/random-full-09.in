250 3306
34 150 18 156 62 150 226 172 168 133 18 150 18 172 249 172 62 156 34 241 18 18 249 18 226 18 18 139 62 249 150 172 156 226 226 150 133 204 204 156 241 139 156 34 133 18 156 62 172 241 133 150 249 34 156 139 232 18 172 150 204 150 34 172 156 232 62 168 226 62 232 34 62 226 62 232 172 172 62 249 18 204 18 204 232 156 62 249 62 133 249 18 34 241 133 249 226 204 139 133 226 62 204 18 150 168 172 34 204 34 241 150 168 204 204 133 133 156 204 249 172 232 226 226 249 204 232 168 62 241 226 150 232 172 34 62 139 232 226 150 18 18 18 150 139 172 62 168 168 34 241 139 150 232 172 156 133 204 241 18 249 241 150 156 172 34 150 249 133 34 62 139 156 150 34 204 172 150 226 62 232 241 62 172 62 172 34 18 139 133 62 226 133 241 18 139 249 232 226 34 156 34 232 168 156 226 156 204 241 249 150 241 34 249 139 62 139 241 18 133 62 168 133 172 156 172 249 34 172 62 241 133 150 226 172 172 18 156 168 249 249 241 18 172 133 62 226 168 249 34
5 32
32 133
32 243
110 133
133 242
110 213
5 164
20 110
32 84
6 84
6 78
105 242
20 188
188 240
138 242
105 165
78 161
42 240
78 200
110 155
155 225
25 213
54 138
6 169
18 161
140 188
23 225
4 110
21 155
75 165
70 138
23 173
149 161
4 126
99 140
5 154
165 249
75 103
32 123
5 67
49 213
154 185
70 143
15 155
18 236
32 104
6 146
95 104
207 225
143 168
120 240
143 198
18 68
99 219
25 245
202 243
131 173
84 102
95 171
20 124
4 233
62 236
21 37
155 158
54 122
81 198
120 208
62 247
68 71
23 241
114 202
177 219
48 202
236 244
57 225
55 99
48 157
70 93
146 201
208 209
65 168
209 218
95 232
7 71
94 173
136 161
62 100
6 76
140 238
105 181
13 208
210 242
172 233
13 147
46 208
5 106
77 114
171 215
39 133
98 122
67 220
42 189
181 224
147 203
165 216
118 210
13 85
11 37
9 118
14 219
50 181
35 76
22 200
168 178
42 69
28 200
16 233
129 203
21 47
106 119
3 126
57 139
161 195
30 84
180 247
56 171
213 217
84 174
181 190
68 205
6 192
60 93
19 177
134 209
122 222
52 71
8 11
8 227
52 117
157 248
19 86
167 240
101 205
88 157
99 128
108 236
218 237
165 235
194 202
26 50
61 220
21 121
207 212
96 192
162 194
2 101
76 87
175 227
31 240
44 99
38 174
12 86
44 231
94 127
59 108
176 198
112 133
186 231
145 186
72 241
54 229
29 178
82 210
77 239
34 146
0 145
16 223
0 83
8 199
63 243
64 229
95 179
66 216
114 226
1 105
67 150
23 45
1 91
118 166
58 59
185 193
29 204
181 221
132 157
90 236
17 139
111 195
138 211
8 183
60 73
187 188
65 125
131 184
79 162
197 198
70 137
11 144
196 210
106 214
153 217
23 159
146 228
120 182
124 246
36 62
99 142
141 204
80 205
43 94
86 163
33 58
40 158
113 162
109 189
97 248
108 152
41 158
156 233
98 115
89 112
10 204
130 184
27 63
51 228
135 201
107 143
92 129
119 191
24 37
170 211
119 230
116 247
206 241
145 160
97 148
153 234
53 127
58 74
139 151
167 186
20 52
33 100
3 50
80 169
62 133
119 228
17 164
38 152
0 138
122 197
25 215
107 188
60 159
225 247
27 133
139 152
64 154
72 237
146 207
142 151
50 227
18 128
20 128
71 99
53 218
105 136
53 147
179 238
51 140
10 164
63 134
132 238
38 114
59 152
93 158
48 86
34 84
98 124
163 192
43 180
6 8
1 215
11 86
47 106
130 179
131 238
97 157
86 146
59 112
71 237
49 137
18 28
51 152
75 212
59 141
149 193
98 221
117 172
83 186
44 122
134 217
34 222
114 137
84 160
27 123
132 217
24 42
39 130
179 194
6 54
9 163
0 12
136 177
57 173
190 205
65 202
36 206
33 41
47 162
91 101
138 196
74 188
114 179
93 245
159 166
6 123
103 107
29 86
51 154
11 213
15 217
150 199
190 233
17 207
69 205
69 90
82 237
69 175
58 177
110 156
83 121
1 71
7 79
93 164
37 44
5 89
50 197
21 34
3 121
40 248
15 222
28 157
125 185
2 159
110 185
92 243
82 242
147 188
140 160
210 240
3 91
19 236
81 240
75 217
92 93
78 184
69 105
97 146
151 237
9 81
119 188
209 242
135 233
22 164
105 248
65 243
10 65
50 171
102 187
222 238
76 153
44 105
56 177
100 249
13 175
34 89
137 206
31 61
114 235
222 231
104 173
183 238
93 140
16 80
134 235
41 196
69 229
16 169
121 180
177 210
21 226
82 132
40 173
72 217
140 143
142 154
59 214
38 218
164 174
86 248
0 224
22 59
199 221
101 133
13 52
91 106
135 243
29 220
2 89
66 234
157 183
59 194
96 124
2 72
91 110
33 92
74 89
12 124
22 109
182 239
213 240
32 194
92 192
216 238
14 23
82 182
100 202
9 52
58 189
25 104
39 115
12 135
98 177
34 238
165 233
11 119
45 224
51 114
150 152
33 246
37 248
22 134
208 229
60 107
58 115
95 177
165 171
60 173
2 29
54 215
14 156
83 207
167 193
104 225
98 205
189 198
42 168
151 246
169 226
155 216
34 72
129 218
179 210
188 215
26 61
140 221
172 217
181 196
9 228
148 177
20 108
13 116
186 232
35 51
33 151
38 83
23 177
50 53
164 240
57 133
236 246
151 222
35 121
11 120
17 138
151 152
137 246
95 138
127 198
52 103
18 169
155 170
24 139
7 34
214 238
113 247
110 122
29 116
83 245
110 119
128 135
57 154
84 166
22 248
12 178
79 189
92 111
39 112
96 139
68 106
105 107
199 218
77 157
152 243
84 198
8 216
63 104
45 56
132 169
158 178
18 37
73 94
142 199
102 217
138 140
40 245
42 65
136 160
63 145
47 199
96 177
36 86
65 105
98 125
30 247
138 147
2 133
16 97
38 156
104 221
99 179
65 204
45 100
75 160
42 227
29 35
32 192
43 225
5 91
100 115
109 192
103 130
138 227
161 223
136 229
75 170
20 31
15 88
175 243
228 242
19 111
21 33
9 67
103 228
70 115
189 194
86 234
35 221
15 43
36 210
115 158
102 126
12 103
50 166
93 216
45 158
185 207
192 195
194 223
155 215
54 120
166 228
28 199
50 144
20 198
16 240
89 159
12 89
52 211
21 211
45 54
34 112
29 100
27 152
84 182
200 212
95 191
30 92
69 134
99 138
2 20
180 205
42 163
42 123
151 209
137 231
107 135
81 192
46 54
116 144
137 244
40 112
172 194
21 95
22 133
66 108
83 88
60 143
93 132
151 166
87 182
3 237
28 96
185 208
52 137
213 214
157 177
200 206
23 50
139 144
151 173
28 164
143 172
34 103
86 140
56 116
137 153
146 160
57 157
65 194
2 184
121 207
105 126
115 116
1 44
31 74
79 215
87 171
41 146
190 209
130 226
75 159
19 53
23 164
168 245
8 129
179 191
209 217
40 144
124 126
142 155
1 197
20 80
69 92
84 126
35 146
125 187
68 136
82 234
39 171
128 248
128 172
34 216
33 107
229 246
28 113
47 181
53 113
39 127
19 48
80 134
4 227
40 117
0 235
54 245
92 165
86 213
109 145
62 206
40 234
179 205
66 220
24 201
40 212
61 87
1 14
81 113
73 144
101 195
101 170
105 172
34 55
57 126
114 201
60 148
1 138
61 143
1 16
50 203
26 98
173 220
143 177
142 235
58 153
33 91
174 179
44 83
74 142
122 130
14 38
36 63
63 207
56 122
164 166
164 208
142 200
5 19
74 227
149 163
197 225
142 175
5 29
24 45
94 219
154 192
7 54
11 45
64 170
63 149
111 180
65 106
25 168
59 104
103 117
54 113
222 243
44 200
59 105
110 171
0 33
181 207
10 171
151 227
127 208
14 87
62 101
36 155
0 154
39 211
96 145
110 201
77 160
36 134
31 86
42 89
41 226
105 223
154 211
210 222
137 208
51 188
29 126
161 219
93 117
45 88
51 76
54 104
43 239
137 230
56 192
161 229
31 60
44 230
173 216
51 217
32 79
191 224
100 106
0 248
94 134
60 177
78 91
192 227
72 233
175 204
62 205
17 145
13 149
211 212
65 110
89 107
83 172
162 220
89 196
146 166
114 176
79 211
174 240
129 227
24 213
11 205
44 52
47 225
16 147
19 77
93 118
64 193
88 140
2 198
68 215
68 235
94 156
64 73
147 248
91 122
60 169
3 104
23 59
53 164
46 149
65 115
79 170
12 226
136 196
8 210
126 151
12 108
81 220
176 221
53 195
58 91
60 232
74 232
80 128
140 168
128 154
167 226
155 194
76 80
131 243
201 230
197 241
39 231
137 147
221 222
2 51
120 148
6 177
58 134
189 207
102 157
23 86
73 84
96 230
76 121
91 238
179 204
126 249
30 216
69 139
80 195
166 219
54 59
181 234
13 73
28 149
169 237
54 103
108 183
93 193
155 236
26 200
128 221
147 249
138 198
142 180
47 205
86 110
4 191
26 246
108 137
17 40
123 205
189 226
157 171
188 224
73 139
19 138
188 198
38 82
72 169
138 238
36 229
166 238
130 205
109 195
138 181
48 123
113 198
64 107
9 159
121 214
60 228
149 210
21 239
23 162
11 31
30 201
41 231
132 194
106 219
226 249
77 122
42 195
40 46
166 213
51 159
142 205
42 230
53 124
49 70
62 171
75 180
31 145
202 203
33 139
121 194
150 169
22 118
4 41
43 211
11 22
90 174
149 227
8 63
81 96
24 159
124 232
4 51
58 201
4 181
52 237
93 232
39 106
4 16
94 119
173 244
121 133
66 113
192 217
216 231
150 218
55 249
128 200
30 106
28 176
62 129
54 148
137 186
34 189
69 76
72 178
42 199
149 228
122 199
72 229
33 204
130 221
4 210
61 182
56 176
154 229
191 200
52 58
140 197
100 207
1 98
20 38
91 198
55 123
48 69
13 212
3 11
52 109
37 48
91 171
28 99
1 57
191 215
74 174
122 185
48 130
87 131
208 237
131 152
111 246
24 222
2 103
169 188
60 135
17 224
15 104
37 57
36 67
111 220
0 80
208 230
118 186
174 208
138 200
66 101
101 229
16 155
168 214
124 242
76 145
93 189
231 247
0 73
56 132
58 180
124 201
88 116
92 188
213 241
116 223
161 208
120 138
39 153
30 207
105 205
17 62
46 187
6 139
11 171
38 175
33 110
66 148
217 245
6 241
43 49
123 154
35 207
99 134
164 181
46 233
104 144
15 230
7 13
48 150
191 231
40 65
15 39
24 89
40 88
110 168
85 178
116 187
58 204
34 50
10 128
77 127
36 174
62 189
16 181
113 181
88 106
19 35
175 196
11 39
198 243
63 96
114 204
7 27
2 127
66 187
78 124
50 90
1 163
170 238
84 183
38 204
9 18
18 114
183 228
210 219
62 244
32 93
83 249
29 45
72 235
215 247
132 191
88 186
83 214
25 143
172 219
167 195
98 222
78 177
17 246
89 185
96 143
32 41
67 84
22 116
52 66
36 140
28 50
76 119
66 79
152 183
40 125
212 248
110 138
47 110
39 57
76 103
9 70
135 173
119 122
51 62
54 140
241 248
138 246
9 247
41 197
156 234
63 107
137 205
70 196
47 202
32 185
129 209
44 196
209 249
44 191
19 216
217 230
208 232
170 235
37 187
9 147
13 213
166 244
107 211
13 150
12 214
4 169
141 229
3 189
13 91
84 209
45 189
95 192
67 208
126 190
122 236
101 224
18 231
42 149
91 213
25 229
155 238
18 222
214 224
6 166
45 71
12 232
50 161
67 143
74 192
87 127
87 141
13 59
14 242
26 72
40 116
76 129
58 152
60 75
147 180
213 246
89 183
104 147
90 164
239 246
176 217
40 169
16 141
36 117
0 7
87 177
102 205
26 65
152 162
6 190
54 160
34 64
61 167
116 127
142 143
179 243
62 168
23 122
153 238
58 78
16 152
37 126
10 197
30 63
48 231
15 100
56 101
166 222
196 198
178 191
35 210
26 202
80 237
21 119
225 238
92 215
48 127
10 64
41 131
27 92
39 151
159 220
107 227
191 208
28 131
154 227
42 132
97 213
119 209
5 178
1 228
116 181
157 207
148 158
157 198
12 223
165 191
86 173
143 175
17 185
45 82
116 246
117 128
86 222
10 25
115 135
0 60
214 236
36 238
170 233
119 220
4 205
41 99
20 225
30 203
87 130
178 188
120 190
111 181
184 191
145 210
49 220
139 221
63 241
112 179
21 96
67 161
167 217
208 215
0 194
0 175
34 131
2 60
97 192
28 153
159 248
124 203
31 236
108 141
117 156
79 247
103 171
135 195
70 163
33 108
65 235
203 214
227 235
174 198
17 119
34 136
169 248
89 116
33 103
43 48
83 216
17 189
188 233
74 106
134 210
118 146
7 150
30 238
127 145
3 33
139 167
26 152
14 226
38 97
17 248
6 45
113 151
53 185
132 159
4 128
42 72
74 150
119 232
165 237
90 249
61 200
40 75
80 179
93 171
22 60
116 191
62 172
76 179
97 239
50 158
8 224
116 218
222 247
75 87
7 223
181 209
96 160
215 242
191 232
152 189
153 191
50 233
61 106
47 237
55 217
69 158
135 154
220 230
143 237
67 103
121 145
115 176
228 243
56 111
214 246
90 107
86 188
135 140
2 8
48 191
7 180
84 96
51 153
188 227
22 244
225 249
74 176
1 182
13 57
85 248
147 158
185 224
122 140
85 147
49 164
166 237
177 204
125 181
70 89
1 47
59 127
32 34
137 207
44 214
63 111
82 240
26 176
8 38
24 216
84 150
12 40
7 168
39 224
93 95
87 99
170 228
115 192
132 134
55 243
49 125
154 219
53 76
76 166
57 176
136 244
162 230
128 233
67 186
31 194
136 221
77 201
179 181
124 191
137 141
19 101
34 79
59 158
112 231
115 247
84 239
146 216
16 117
190 207
75 137
20 28
157 241
107 209
44 167
14 188
36 57
113 184
91 247
31 33
178 204
40 182
96 181
73 202
12 182
224 235
127 130
36 136
195 217
128 179
202 242
23 52
97 198
5 138
31 124
3 87
145 181
153 212
200 207
59 232
40 71
21 94
44 101
123 188
11 226
58 228
146 155
125 199
135 170
64 171
13 26
109 209
24 211
60 152
119 128
55 101
45 173
229 241
178 181
40 86
18 72
80 188
95 240
106 115
63 72
66 249
15 238
20 140
83 197
149 155
61 210
94 236
136 191
93 94
118 194
138 169
140 232
18 227
80 116
51 215
31 216
201 233
119 222
159 196
177 229
16 174
32 48
19 41
35 98
0 165
11 23
205 245
6 133
57 196
108 169
77 162
133 144
155 249
126 242
0 204
42 214
120 245
16 129
16 87
170 179
117 197
140 151
118 120
224 241
172 175
27 111
117 125
195 199
78 126
30 110
87 124
177 245
92 144
182 183
1 79
66 232
134 245
166 211
28 181
58 104
119 213
196 200
88 245
107 123
12 139
95 163
60 202
218 242
43 195
118 173
86 101
25 170
50 91
70 216
76 142
21 215
13 41
119 192
202 211
32 179
111 201
119 205
94 216
76 163
7 80
140 146
33 37
0 110
39 98
51 184
37 191
39 122
125 152
95 106
111 137
201 248
87 119
33 125
24 212
108 242
183 190
193 227
120 201
43 157
105 240
47 189
18 97
74 182
31 38
195 221
20 90
1 155
151 197
90 230
149 152
27 249
164 237
108 212
85 240
88 112
32 229
119 149
7 195
181 197
109 147
117 176
233 242
26 233
64 203
130 153
83 211
177 241
44 202
117 163
65 68
17 98
1 99
39 230
107 129
71 90
34 198
166 204
64 202
28 93
26 154
73 115
0 1
112 197
123 233
53 198
152 167
172 192
122 217
63 109
187 237
115 240
77 179
46 171
31 81
81 93
130 219
12 195
138 229
89 122
23 63
116 214
20 23
45 81
18 33
141 180
59 169
38 62
171 202
76 243
15 172
98 166
157 224
1 87
97 246
37 49
15 70
225 233
85 168
175 185
68 224
169 249
58 167
14 70
17 111
71 143
111 146
56 129
203 226
104 127
42 155
14 163
153 221
8 65
86 233
54 222
98 207
99 236
178 179
103 234
180 221
161 226
4 101
23 200
9 220
22 80
193 209
18 82
58 132
12 106
115 163
25 179
79 221
124 208
83 196
82 105
231 238
204 207
103 153
28 74
3 32
82 247
64 161
5 54
107 177
158 217
8 135
50 178
207 221
19 145
24 73
182 191
181 212
33 70
44 229
32 244
20 58
45 157
56 65
13 54
32 54
109 137
44 147
94 128
83 246
1 109
130 157
2 236
61 243
40 191
50 224
20 230
99 210
81 127
25 119
60 112
34 249
106 222
142 192
20 219
175 183
49 177
20 147
7 62
202 221
175 202
46 243
106 180
28 71
36 160
28 174
134 234
64 201
74 165
4 188
44 118
22 44
8 127
91 183
97 184
54 126
19 135
127 188
40 135
55 205
25 223
29 160
116 203
15 210
147 247
106 232
20 235
52 140
24 136
60 86
8 122
167 218
94 209
109 121
49 86
10 30
114 248
92 148
115 122
27 110
41 221
54 135
33 109
69 206
126 152
164 185
148 245
97 107
88 91
134 153
211 225
14 211
0 34
36 188
199 207
9 211
34 132
5 56
196 236
119 237
123 185
112 169
25 188
37 152
197 230
146 162
72 124
20 136
138 185
54 92
177 221
100 154
141 211
70 123
85 155
87 247
70 77
147 234
52 247
30 246
139 205
117 166
5 171
146 179
79 182
178 205
39 100
142 218
2 177
31 173
101 208
54 223
34 83
76 170
9 235
69 242
153 249
32 170
82 217
110 246
130 161
52 127
168 225
56 183
38 64
83 101
118 156
157 191
18 35
72 80
22 183
47 78
151 220
113 171
24 36
147 168
51 148
1 64
213 249
32 121
5 31
84 190
73 157
180 226
125 248
111 216
188 193
85 246
44 74
6 72
94 178
60 221
147 190
105 190
119 226
72 246
38 183
118 126
132 143
8 182
173 240
22 172
137 155
24 182
155 226
113 175
114 196
14 96
60 153
21 99
41 230
210 244
121 200
104 150
93 222
71 227
49 111
18 24
19 205
157 205
102 163
17 206
43 220
5 233
46 119
25 203
72 115
122 226
123 132
77 178
151 234
9 27
127 225
166 225
26 109
2 31
163 190
14 146
133 221
161 217
1 88
17 126
3 125
30 146
13 114
31 241
144 177
141 233
21 203
71 114
104 196
78 146
19 75
13 190
18 201
195 196
32 103
24 131
28 190
29 135
3 211
72 137
25 39
29 222
32 134
3 35
141 242
69 220
130 176
89 132
120 231
44 220
113 119
66 86
26 231
10 232
126 247
41 216
68 244
164 202
10 155
26 119
104 148
63 218
88 178
146 205
73 214
173 227
63 125
0 5
79 85
108 162
160 200
61 67
180 227
15 81
5 68
48 170
182 225
71 166
11 194
45 64
31 59
139 237
125 163
19 231
56 205
130 172
122 180
73 122
93 172
68 78
52 87
16 210
108 148
1 34
163 194
117 161
32 167
66 82
94 165
5 120
114 212
101 233
199 240
8 181
67 201
73 167
50 156
28 73
103 114
110 248
53 128
7 57
219 249
94 146
38 81
85 181
62 74
66 114
61 248
37 246
38 57
75 196
179 193
143 176
236 248
143 244
94 244
89 149
39 51
52 92
67 92
27 96
47 210
2 145
89 141
20 161
12 165
7 42
181 249
154 177
117 194
199 236
18 40
25 219
2 40
93 173
199 244
101 121
68 154
64 211
57 145
118 218
63 227
50 242
2 137
15 157
40 192
61 230
145 239
45 65
61 104
40 126
94 223
75 105
23 125
8 209
74 205
91 209
80 159
188 211
182 201
11 85
174 241
178 196
184 229
138 195
169 235
228 248
95 244
6 44
8 137
24 153
34 191
160 167
96 147
14 239
127 148
14 209
83 97
54 191
1 140
17 68
204 208
60 215
36 167
59 203
55 139
133 222
191 216
68 200
2 95
42 55
25 227
16 105
113 220
29 50
177 236
51 108
110 161
7 75
76 123
37 206
47 112
102 134
69 240
166 220
17 191
1 198
155 228
103 246
85 132
152 217
99 156
68 131
85 233
130 170
87 226
189 212
206 210
54 180
191 206
59 111
163 173
119 177
20 239
79 161
128 206
49 238
13 95
0 21
9 71
110 165
84 165
113 214
12 66
63 197
136 205
86 226
43 74
30 56
138 241
163 244
3 39
114 157
81 212
5 71
162 209
39 95
46 194
227 246
170 187
127 193
97 175
45 191
88 100
215 226
7 24
113 115
118 199
196 218
53 229
19 64
134 221
89 210
190 222
22 90
7 193
96 187
1 172
19 106
20 231
69 146
32 172
105 208
196 203
98 235
171 234
53 152
25 81
30 234
102 222
148 151
12 30
55 81
71 215
143 182
44 140
160 209
90 210
37 40
204 230
177 211
36 169
110 125
187 213
58 246
42 145
17 107
80 123
19 242
23 144
33 39
3 65
57 146
42 111
19 238
58 73
162 213
31 109
31 52
34 147
116 207
113 172
82 203
24 223
14 220
123 212
70 197
143 200
25 47
17 25
31 207
97 127
47 168
18 243
210 236
70 125
14 221
11 206
141 200
130 147
57 127
81 236
17 172
19 89
48 245
193 201
2 230
31 40
28 76
177 199
62 229
84 233
32 39
31 100
104 156
83 99
118 157
170 189
74 91
6 189
126 127
6 108
24 47
136 234
27 227
85 187
8 20
56 152
9 201
14 237
71 144
36 149
114 168
39 60
91 248
131 195
93 100
66 131
12 53
205 214
153 247
89 178
158 175
19 234
94 206
129 139
140 182
147 150
36 230
12 111
80 110
105 115
145 150
162 184
189 236
13 198
203 243
157 217
40 148
10 34
19 188
47 74
97 238
94 227
88 120
107 182
13 218
31 96
158 193
46 242
3 194
50 108
6 86
51 94
51 174
48 107
75 134
96 193
64 172
99 149
23 223
43 78
114 221
2 164
153 171
75 97
46 83
52 130
136 159
72 164
58 63
26 85
61 91
151 189
34 179
165 202
51 73
173 195
115 245
90 172
94 151
36 194
113 229
158 164
38 248
61 237
24 160
60 214
204 215
129 230
46 56
54 193
70 181
7 182
54 97
33 138
177 216
6 194
2 70
52 96
33 202
47 103
3 165
16 40
1 191
114 155
36 159
40 82
26 83
205 240
232 235
25 177
73 159
120 186
15 114
19 218
26 237
137 151
110 216
201 209
16 185
83 215
99 109
142 240
136 158
13 167
2 139
197 227
97 216
9 117
103 240
0 217
43 150
177 182
20 105
6 229
7 89
128 166
92 158
106 243
15 125
57 75
159 191
109 241
3 82
174 194
117 232
72 234
56 160
63 247
10 146
39 44
142 204
47 245
119 194
22 131
54 95
106 220
61 174
108 221
68 176
72 147
22 173
47 131
84 89
98 223
15 36
105 150
81 193
107 139
135 168
81 148
120 233
21 144
78 227
5 194
3 66
161 197
129 187
176 229
239 244
2 187
21 150
19 117
23 149
74 78
137 235
167 242
233 247
134 187
225 235
118 238
77 234
18 223
9 59
166 173
127 128
40 104
164 177
102 137
141 238
137 236
84 230
146 161
49 240
5 188
103 207
84 179
80 95
223 229
113 122
173 207
28 29
96 166
157 175
63 194
134 205
13 74
2 23
120 136
21 131
116 152
83 104
120 196
31 198
142 190
24 51
165 226
38 227
26 144
2 82
53 106
55 210
55 227
55 82
217 231
136 230
10 134
25 151
104 163
131 157
48 193
30 248
58 75
94 147
147 221
239 241
121 174
100 137
38 209
9 106
64 81
111 124
8 167
46 133
123 193
115 182
32 101
98 164
28 194
167 200
46 60
100 226
0 172
66 176
192 209
83 188
164 214
121 124
20 159
75 188
40 183
45 73
113 177
68 173
39 126
110 147
71 148
188 228
73 134
104 182
75 204
88 104
2 5
27 186
9 180
117 230
59 179
88 115
134 171
2 48
171 177
219 239
52 57
111 141
17 188
75 183
30 243
95 130
29 236
107 186
3 19
93 194
127 162
61 246
77 195
42 160
134 237
66 129
3 137
169 204
35 246
109 211
55 100
26 165
3 117
227 240
74 211
6 56
3 58
20 119
151 182
54 186
25 90
87 212
17 175
73 187
63 209
59 117
79 155
5 238
16 236
227 230
26 149
57 162
150 235
9 243
82 168
37 240
12 112
0 181
210 215
109 212
125 132
30 122
124 151
97 202
208 223
199 241
112 224
60 183
126 158
91 218
128 227
172 234
4 129
115 189
162 212
127 206
196 247
68 204
82 166
57 116
69 116
115 149
103 231
186 201
84 123
28 145
195 211
137 154
210 249
16 225
13 227
107 149
132 141
109 200
18 27
5 201
41 147
48 72
7 12
142 179
39 121
3 46
226 229
19 241
43 247
167 170
202 225
19 163
46 192
36 43
43 193
212 228
140 166
19 191
200 203
10 79
61 216
100 121
11 109
118 179
89 180
110 137
213 233
69 75
95 154
137 163
134 204
147 228
99 184
60 124
111 127
62 209
18 95
31 75
138 141
21 69
96 212
20 92
84 115
73 104
172 225
31 242
80 194
92 161
166 233
211 224
73 205
121 246
67 135
59 134
146 239
120 204
193 208
2 98
54 230
92 146
98 247
136 180
65 120
35 91
70 71
57 105
52 182
168 219
16 145
8 171
54 181
36 184
92 241
105 152
73 140
39 74
60 92
15 197
27 210
89 199
166 192
23 178
68 98
96 167
163 168
97 130
6 212
117 242
29 127
109 166
4 47
96 102
32 127
109 229
50 157
47 70
61 153
65 127
17 217
203 225
210 230
79 176
132 218
0 149
123 133
2 63
1 103
74 119
144 232
109 239
141 210
186 190
110 175
177 232
131 229
174 203
97 212
150 236
70 106
64 209
100 124
149 180
128 153
1 60
122 158
59 230
8 109
108 245
73 220
9 89
10 145
162 248
29 221
100 176
176 202
153 216
180 213
42 110
146 148
111 227
63 94
43 221
104 135
32 227
118 167
205 221
31 159
174 238
66 115
55 166
2 161
1 36
162 203
53 55
43 174
118 214
135 221
26 99
114 178
213 232
10 180
43 155
28 232
160 161
115 144
114 130
164 223
130 196
66 73
49 141
24 125
208 227
29 174
36 187
43 68
42 232
125 141
141 220
200 245
62 108
64 242
123 207
35 108
101 117
33 221
91 199
5 160
18 126
54 74
73 83
38 181
133 170
35 82
78 215
140 240
231 237
148 217
63 71
63 128
39 48
47 236
103 195
223 227
32 70
36 59
61 231
19 51
126 230
169 179
11 210
31 197
90 139
157 216
22 213
146 167
130 195
19 219
199 245
139 212
7 166
161 230
36 55
168 194
100 122
82 246
105 128
83 123
42 243
111 149
59 115
48 169
84 147
178 241
80 176
138 165
82 117
71 91
74 244
57 159
94 217
70 172
50 154
155 186
87 241
163 219
2 150
19 130
14 26
52 242
231 244
103 198
30 230
24 177
64 84
180 186
36 83
93 131
61 139
106 140
73 178
52 191
15 87
157 190
42 95
140 167
87 123
43 110
148 182
10 102
37 125
175 240
22 166
42 231
77 194
141 151
41 148
109 194
117 227
138 213
36 211
69 110
25 62
170 182
57 88
71 206
103 179
189 247
117 241
105 153
144 151
174 221
61 142
112 229
61 96
169 202
17 147
166 214
98 118
14 198
38 113
37 224
171 191
95 198
82 197
15 95
144 243
14 155
175 228
2 200
16 232
6 59
171 230
21 22
111 198
172 212
116 206
19 151
160 223
211 246
27 159
155 211
124 212
121 248
158 199
124 164
183 239
128 187
123 217
151 160
37 143
126 218
122 179
92 245
164 227
72 196
11 230
52 230
80 184
116 175
68 191
57 205
28 124
140 231
56 234
94 177
18 233
17 64
162 183
11 175
13 141
84 132
120 207
149 189
4 78
122 142
94 170
72 173
109 168
4 178
227 232
38 211
106 221
186 213
137 211
115 222
217 235
70 203
5 229
96 227
15 93
154 160
79 199
48 166
34 148
110 186
45 225
65 81
40 166
45 59
16 93
162 172
5 241
79 196
92 141
21 216
115 230
175 215
82 106
179 196
87 142
116 238
36 96
76 118
123 243
67 157
83 191
29 71
30 32
119 143
75 136
69 195
183 229
161 172
25 224
144 228
151 207
40 78
34 161
168 231
108 228
130 158
151 215
4 201
0 136
67 91
31 214
18 98
28 60
146 210
156 204
19 59
137 150
99 143
117 153
85 136
227 245
235 236
137 180
210 223
159 235
16 32
51 205
73 223
46 129
205 247
3 124
122 193
157 186
167 244
12 60
24 210
105 134
99 239
38 76
35 64
182 200
9 134
65 75
18 193
156 193
25 84
91 239
116 201
15 44
26 121
125 134
86 227
85 133
65 242
0 182
26 170
0 53
24 238
47 91
125 244
199 246
108 115
209 229
26 40
49 77
142 195
98 213
118 227
124 225
63 130
18 197
72 106
23 239
179 221
50 56
127 133
130 237
111 158
25 167
3 245
118 213
65 198
16 227
72 181
77 84
64 174
41 138
32 50
46 153
124 237
70 127
8 39
157 218
175 244
150 157
114 116
19 37
34 165
100 211
9 184
140 179
61 225
108 123
9 167
40 87
84 156
73 76
107 117
206 207
44 186
116 221
14 229
64 165
4 20
64 224
117 244
117 231
23 230
25 101
20 155
79 166
39 43
59 124
34 149
106 174
176 232
106 183
23 131
5 82
28 65
216 235
58 192
221 228
174 195
95 122
14 17
15 181
15 154
123 195
56 57
179 203
175 223
32 86
139 179
156 212
133 206
3 197
98 201
95 159
158 169
19 179
105 148
15 26
80 120
177 217
4 198
132 241
41 229
2 213
7 143
45 214
134 172
194 249
80 119
57 113
47 234
126 217
3 227
43 218
153 205
87 215
1 244
124 162
63 108
6 237
92 145
168 189
19 171
1 12
68 158
26 82
123 229
68 233
39 139
99 117
55 72
155 203
83 145
210 211
36 106
3 8
81 102
123 169
21 45
159 180
19 187
14 102
46 47
18 107
54 102
5 144
101 153
83 108
18 93
27 179
155 247
63 151
67 70
52 99
33 182
6 195
120 184
81 200
18 106
91 132
