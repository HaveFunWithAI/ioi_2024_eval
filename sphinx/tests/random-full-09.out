250 2149
34 150 18 156 62 150 226 172 168 133 18 150 18 172 249 172 62 156 34 241 18 18 249 18 226 18 18 139 62 249 150 172 156 226 226 150 133 204 204 156 241 139 156 34 133 18 156 62 172 241 133 150 249 34 156 139 232 18 172 150 204 150 34 172 156 232 62 168 226 62 232 34 62 226 62 232 172 172 62 249 18 204 18 204 232 156 62 249 62 133 249 18 34 241 133 249 226 204 139 133 226 62 204 18 150 168 172 34 204 34 241 150 168 204 204 133 133 156 204 249 172 232 226 226 249 204 232 168 62 241 226 150 232 172 34 62 139 232 226 150 18 18 18 150 139 172 62 168 168 34 241 139 150 232 172 156 133 204 241 18 249 241 150 156 172 34 150 249 133 34 62 139 156 150 34 204 172 150 226 62 232 241 62 172 62 172 34 18 139 133 62 226 133 241 18 139 249 232 226 34 156 34 232 168 156 226 156 204 241 249 150 241 34 249 139 62 139 241 18 133 62 168 133 172 156 172 249 34 172 62 241 133 150 226 172 172 18 156 168 249 249 241 18 172 133 62 226 168 249 34
